boards/adafruit/qt_py_esp32s3/adafruit_qt_py_esp32s3_appcpu.dts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024 Ian Wakely
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32s3/esp32s3_wroom_n8.dtsi>
+#include "adafruit_qt_py_esp32s3-pinctrl.dtsi"
+#include <espressif/partitions_0x0_amp.dtsi>
+
+/ {
+	model = "Adafruit QT Py ESP32S3 APPCPU";
+	compatible = "espressif,esp32s3";
+
+	chosen {
+		zephyr,sram = &sram0;
```

Comment:
Please, change to sram1 to take in account #80340

Reply:
Addressed.